geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/rules/tests/SharedCountersRuleDistributedTest.java

```diff
@@ -147,6 +151,60 @@ public void inc_multipleThreads_fromEveryVM_getTotal_returnsExpectedTotal() thro
     assertThat(sharedCountersRule.getTotal(ID)).isEqualTo(expectedIncrements);
   }
 
+  @Test
+  public void addVmDoesNotFail() {
```

Comment:
The test name doesn't reflect the main idea, which is that the counter rule establishes the right initial value.